cmds/ls/ls.go

```diff
@@ -67,8 +73,11 @@ func listName(d string, w io.Writer, prefix bool) error {
 			}
 		}
 
-		// Print the file in the proper format.
-		fmt.Fprintln(w, stringer(fi))
+		// Hide .files unless -a was given
```

Comment:
. file issues should be a separate PR. Not all agree that hiding . files was a good idea.
https://plus.google.com/+RobPikeTheHuman/posts/R58WgWwN9jp
The question, should we continue this mistake? Is there harm in just printing .files?I don't see a problem.

Reply:
This PR has two commits to separate this. I don't think a separate PR is necessary.